containers/docs/set.rst

```diff
@@ -24,7 +24,7 @@ provided by the ``containers`` package: :haddock:`/Data.Set` and
 
 
 All of these implementations are *immutable* which means that any update
-functions do not modify the set that you passed in, they creates a new set. In
+functions do not modify the set that you passed in, they create a new set. In
 order to keep the changes you need to assign it to a new variable. For example::
```

Comment:
While you're at it, that comma should probably be a semicolon, since it begins an independent clause.